server/crawler/crawler.go

```diff
@@ -773,7 +777,58 @@ func extractDockerComposePackageContent(
 		"",
 		0,
 		nil,
-	), true, nil
+	)
+
+	dockerComposeEnvFilePath := path.Join(packageRepositoryMetadata.RootPath, dockerComposeEnvFilename)
+
+	envFileContentResult, _, resp, err := client.Repositories.GetContents(ctx, packageRepositoryMetadata.Owner, packageRepositoryMetadata.Name, dockerComposeEnvFilePath, repoGetContentOpts)
+	if err != nil {
+		return nil, false, stacktrace.Propagate(err, "an error occurred retrieving the docker compose env file")
+	}
+	if resp.StatusCode == http.StatusNotFound {
+		logrus.Warnf("Docker compose repository '%s' is missing an env file", packageRepositoryMetadata.Name)
+		return packageContent, true, nil
+	}
+
+	envFileContentArguments, err := extractArgumentsFromEnvFileContent(envFileContentResult)
```

Comment:
An enhancement would be to extract the list of arguments from the docker compose file and augment them with the env file default values.  That way we are getting something valuable even when the env file is non existent.